draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1372,10 +1373,6 @@ The following Disclosures are created by the Issuer:
 
 {{examples/arf-pid/disclosures.md}}
 
-The following decoy digests are added:
-
-{{examples/simple_structured/decoy_digests.md}}
```

Comment:
Note that this was pointing to the wrong file...

Reply:
why pointing to a correct file not an option?

Reply:
This example doesn't use decoys anymore (probably shouldn't have in the first place) so there's no decoy digests file content to point to. #337 was extremely confusing to get to the bottom of due in large part to including the decoy_digests.md from a different example.